### PR TITLE
Alerts guide fixes from #78

### PR DESCRIPTION
**Pull request by adnanrahic**

### CODE REVIEW

**Comment by adnanrahic** [comment]
You only need to read the `alerts-guide.md` nothing else.

File: docs/guide/alerts-guide.md [hunk 1]
```diff
@@ -37,21 +37,21 @@ Default Alerts for a Monitoring App:
 - **Heartbeat Alert** if the **Agent stops sending data to the Monitoring App**
 - **Metric Anomaly Alert** for misbehaving Nodes
 
-> In this example which is a Elasticsearch Monitoring App, the Java usage threshold and Elasticsearch Node Anomalies are integration-specific default Alerts.
+In this example which is an Elasticsearch Monitoring App, the Java usage threshold and Elasticsearch Node Anomalies are integration-specific default Alerts.
 
 ![Sematext Cloud System Generated Alerts](https://sematext.com/docs/images/guide/alerts-and-events/system-generated-alerts.png "Sematext Cloud System Generated Alerts")
 
-The [Alert Rules](https://apps.sematext.com/ui/events/alerts/rules) page is where you will find all default, and created, Alerts for all Apps you'ce created. Here you can toggle, edit or delete any existing Alerts.
+The [Alert Rules](https://apps.sematext.com/ui/events/alerts/rules) page is where you will find all default and created, Alerts for all Apps you've created. Here you can toggle, edit or delete any existing Alerts.
```
Comment (otisg) [inline docs/guide/alerts-guide.md]: "page is where you will find all default and created, Alerts for all Apps you've created" sounds strange, something is a little extra there...

File: docs/guide/alerts-guide.md [hunk 2]
```diff
@@ -37,21 +37,21 @@ Default Alerts for a Monitoring App:
...
 
-![Sematext Cloud Alerts Rules Window](https://sematext.com/docs/images/guide/alerts-and-events/alert-rules-window.png "Sematext Cloud Sematext Cloud Alerts Rules Window")
+![Sematext Cloud Alerts Rules Window](https://sematext.com/docs/images/guide/alerts-and-events/alert-rules-window.png "Sematext Cloud Alerts Rules Window")
 
 ## Creating Alerts
 
-Sematext Alerts can cover both complicated alerting scenarios with multiple rules on both Logs and Monitoring, as well as Events. A basic Alert Policy with an email or Slack notification is enough to cover elementary alerting needs.
+Sematext Alerts can cover both complicated alerting scenarios with multiple rules on both Logs and Monitoring, as well as Events. A basic Alert Rule with an email or Slack notification is enough to cover elementary alerting needs.
```
Comment (otisg) [inline docs/guide/alerts-guide.md]: that "on" in "rules on both" sounds strange.
Maybe "rules for both..."?

File: docs/guide/alerts-guide.md [hunk 3]
```diff
@@ -68,28 +68,19 @@ A **Save Query** window will open with the option to **Enable Alert**. When it's
...
+  <source src="../../images/guide/alerts-and-events/alert-scheduling.mp4" type="video/mp4" />
+</video>
+
 
 Press Save and you're done. Check out the [integrations](https://sematext.com/docs/integration/) documentation for more information about alert notification hooks.
 
 ### Alerting on Metrics
 
-Metrics have both **Threshold** and **Anomaly** Alerts, but a Monitoring App has **component-based Alerting**. Every time-series metric is a _component_. Alerts can be created for each component, just as they can be added to custom [dashboards](./dashboards-guide/).
+Monitoring Apps have **metric-based Alerting**. Metrics have both **Threshold** and **Anomaly** Alerts which can be created for each metrics component, just as they can be added to custom [dashboards](./dashboards-guide/).
```
Comment (otisg) [inline docs/guide/alerts-guide.md]: "Alerts which can be created for each metrics component, just as they can be added to custom dashboards" sounds incorrect.
Alerts in Monitoring Apps are created on specific metrics.  They have nothing to do with components or dashboards.

Comment (adnanrahic) [reply docs/guide/alerts-guide.md]: I've re-worded this to be more concrete.

File: docs/guide/alerts-guide.md (hunk 2 above)
Comment (adnanrahic) [reply docs/guide/alerts-guide.md]: fixed

File: docs/guide/alerts-guide.md (hunk 1 above)
Comment (adnanrahic) [reply docs/guide/alerts-guide.md]: Reworded this to sounds better.

**Pull request merged**